<ui version="4.0" >
 <class>vvDeformationDialog</class>
 <widget class="QDialog" name="vvDeformationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Deformation field computation</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Input Sequence</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="inputSequenceBox" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="refImageLabel" >
     <property name="text" >
      <string>Reference Image (0)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="refImageSlider" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Maximum number of iterations       </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="iterSpin" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>1000000</number>
     </property>
     <property name="singleStep" >
      <number>10</number>
     </property>
     <property name="value" >
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Number of consecutive useless iterations before stop</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="stopSpin" >
     <property name="maximum" >
      <number>100000</number>
     </property>
     <property name="value" >
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Alpha</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QDoubleSpinBox" name="alphaSpin" >
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QDoubleSpinBox" name="sigmaSpin" >
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Number of threads</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QSpinBox" name="threadSpin" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>99</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Ouput file name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="outputLineEdit" >
       <property name="text" >
        <string>output_vf.mhd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputPushButton" >
       <property name="text" >
        <string>Chose file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vvDeformationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vvDeformationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel" >
     <x>219</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
